zhaquirks/lixee/zlinky.py

```diff
@@ -186,3 +195,15 @@ class ZLinkyTIC(CustomDevice):
             },
         },
     }
+
+
+class ZLinkyTICFWV12(ZLinkyTIC):
+    """ZLinky_TIC from LiXee with firmware v12.0+."""
+
+    signature = deepcopy(ZLinkyTIC.signature)
+
+    # Update model info
+    signature[MODELS_INFO] = [(LIXEE, "ZLinky_TIC_FW_V12")]
```

Comment:
Should I keep the same model name so that the fix in https://github.com/home-assistant/core/pull/82602/files#diff-97adf01229931f17610655838232182c16a7f10291291d780cfd472be187406eR510 continues to work?

Reply:
But, is `ZLinky_TIC_FW_V12` the model value for that devices, isn't it?
You need to keep the model to be able to match the quirk.
I believe that you can overwrite it in the replacement part, but not sure if would be a good thing.
You can also add the `ZLinky_TIC_FW_V12` model in HA to expose all the entities.

Reply:
I invented it. The quirk works with both strings for the model version. I believe it is only esthetic.

Reply:
But, you're replacing the `MODELS_INFO` in the new quirk. No device could match the signature...
Do you have check if v12 devices are loading the quirk? From what I see them couldn't.

Reply:
(And would be fine having 2 quirks with the same `MODELS_INFO` and diferent signatures)

Reply:
Ok, got it. I did not actually test to remove then re-add my device ZLinky_TIC with firmware v12, that's why I did not face any issue.
`model` attribute of `basic` cluster returns `ZLinky_TIC`, so I'll revert that change.